csi/controller.go

```diff
@@ -123,6 +110,27 @@ func (s *OsdCsiServer) ControllerUnpublishVolume(
 	return nil, status.Error(codes.Unimplemented, "This request is not supported")
 }
 
+// ControllerGetVolume is a CSI API which implements getting a single volume
+func (s *OsdCsiServer) ControllerGetVolume(
```

Comment:
Add a comment here about security and how this call can only be used through a unix domain socket by the controller, and that it is a read only request.

Reply:
Added